db/migrate/20231201164130_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.text :photo
```

Comment:
- [ ] please kindly use the example below for the datatype of the `:photo`, rather than using `t.text` # as photo should be a link to an image not a file

- code snippet
```ruby
 t.string :photo
```